qa/src/test/java/org/eclipse/kapua/service/StepData.java

```diff
@@ -0,0 +1,45 @@
+/*******************************************************************************
+ * Copyright (c) 2011, 2017 Eurotech and/or its affiliates and others
```

Comment:
I know this is only a minor thing ... but this file seems to be created in 2017 and thus the correct copyright header line would be "Copyright (c) 2017 Eurotech…"